<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseEmojiDlg</class>
 <widget class="QDialog" name="ChooseEmojiDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>选择要插入的表情</string>
  </property>
  <widget class="QLabel" name="hintLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ChooseEmojiWidget" name="chooseWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/default/res/face/page.bmp);</string>
   </property>
  </widget>
  <widget class="QLabel" name="gifLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChooseEmojiWidget</class>
   <extends>QLabel</extends>
   <header>chooseemojiwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
